### ISSUE TITLE
0.3pre: Keyboard language autodetection is missing

### ISSUE THREAD

**Issue by probonopd** [mszoek/airyx#101]
Since airyx is based on helloSystem, I was hoping that the keyboard language, system language, and timezone would be automatically set like it is the case in helloSystem when you use a compatible keyboard (currently: Raspberry Pi keyboards) by way of `/usr/local/bin/localize`. Unfortuately it is not working, resulting in an English system with English keyboard even though a German Raspberry Pi keyboard is attached.

**Comment by mszoek** [comment]
Ah yes, my apologies for this one! I ripped out most of start-hello to build my own startDesktop script because I have not ported all the pieces over yet and not all the commands there are relevant with kwin/plasma. Am slowly bringing pieces back over. We can definitely add this bit back in for next release!

**Comment by probonopd** [comment]
This would be cool. Having the keyboard language, system language, and timezone auto-guessed is almost always better than just plain assuming English for everyone, imho.

**Comment by mszoek** [comment]
Agreed! :100: I like the way Apple handled unknown keyboards as well, asking you to hit "the key next to Shift"  and so on to figure out the layout. That's a good fallback.
